components/apimgt/org.wso2.carbon.apimgt.internal.service/src/main/java/org/wso2/carbon/apimgt/internal/service/impl/ApisApiServiceImpl.java

```diff
@@ -94,6 +95,7 @@ public Response deployedAPIRevision(List<DeployedAPIRevisionDTO> deployedAPIRevi
         APIProvider apiProvider = RestApiCommonUtil.getLoggedInUserProvider();
 
         List<String> revisionUUIDs = new ArrayList<>();
+        String tenantDomain = CarbonContext.getThreadLocalCarbonContext().getTenantDomain();
```

Comment:
we can use APIInfo getAPIInfoByUUID(String id) throws APIManagementException  in APIProvider from that get the organization id